enterprise-beans/4.0/_index.md

```diff
@@ -1,19 +1,21 @@
 ---
-title: "Jakarta Enterprise Beans 4.0 (under development)"
-date: 2020-mm-dd
+title: "Jakarta Enterprise Beans 4.0 (milestone)"
+date: 2020-06-12
 summary: "Release for Jakarta EE 9"
 ---
 
 Jakarta Enterprise Beans defines an architecture for the development and deployment of component-based business applications.
 
 * [Jakarta Enterprise Beans 4.0 Release Record](https://projects.eclipse.org/projects/ee4j.ejb/releases/4.0/plan)
   * [Jakarta Enterprise Beans 4.0 Release Plan](https://github.com/eclipse-ee4j/ejb-api/blob/master/4.0-PLAN.adoc)
-* [Jakarta Enterprise Beans 4.0 Specification Document]() (PDF)
-* [Jakarta Enterprise Beans 4.0 Specification Document]() (HTML)
-* [Jakarta Enterprise Beans 4.0 Javadoc](./apidocs)
+* [Jakarta Enterprise Beans, Core Features 4.0 Specification Document](./enterprise-beans-spec-core-4.0.pdf) (PDF)
```

Comment:
I would suggest renaming these files to indicate they are not "final".  The current name could suggest that this is the final 4.0 spec.  I know that the header page indicates that this is a "public draft", but you have to look for that to know (not intuitive).  Another idea that many projects are doing is to incorporate the pom version into the name of the generated spec.  You can look at the Platform specs (platform, web profile, or managed beans) for an example.

Reply:
I have renamed the files with "-public-draft". I feel the status is more appropriate than the pom version. For e.g., the final version of the specification can have "-final" instead of no qualifier if we follow the pom version.